core/ir/aarch64/codec.c

```diff
@@ -1039,6 +1039,33 @@ encode_opnd_s_const_sz(uint enc, int opcode, byte *pc, opnd_t opnd, OUT uint *en
     return false;
 }
 
+/* cmode4_b_sz : Operand for byte elements' shift amount
+ */
+static inline bool
+decode_opnd_cmode4_b_sz(uint enc, int opcode, byte *pc, OUT opnd_t *opnd)
+{
+    /* cmode size amounts
+     * 1110  8    0
+     * 10x0  16   0,8
+     * 0xx0  32   0,8,16,24
```

Comment:
Yes, I put the whole `cmode` table comment in all the `cmode` handlers. I take your point it can be confusing. Removed entries in this and other `cmode` handlers.

Also omitted to check `cmode` encoding for byte elements because the `codec.txt` definition was out-of-date :-( Now fixed.